src/test/java/org/zaproxy/zest/test/v1/ZestClientElementMouseOverUnitTest.java

```diff
@@ -0,0 +1,105 @@
+import org.zaproxy.zest.core.v1.ZestClientElementMouseOver;
+import org.zaproxy.zest.core.v1.ZestClientLaunch;
+import org.zaproxy.zest.core.v1.ZestJSON;
+import org.zaproxy.zest.core.v1.ZestScript;
+import org.zaproxy.zest.impl.ZestBasicRunner;
+
+/** Unit test for {@link ZestClientElementMouseOver}. */
+class ZestClientElementMouseOverUnitTest extends ServerBasedTest {
+
+    private static final String PATH_SERVER_FILE = "/test.html";
+
+    @Test
+    void shouldNotBePassive() {
+        // Given / When
+        ZestClientElementMouseOver clientElementMouseOver = new ZestClientElementMouseOver();
+        // Then
+        assertEquals(false, clientElementMouseOver.isPassive());
+    }
+
+    @Test
+    void shouldNotHaveWindowHandleByDefault() {
+        // Given / When
+        ZestClientElementMouseOver clientElementMouseOver = new ZestClientElementMouseOver();
+        // Then
+        assertNull(clientElementMouseOver.getWindowHandle());
+    }
+
+    @Test
+    void shouldBeEnabledByDefault() {
+        // Given / When
+        ZestClientElementMouseOver clientElementMouseOver = new ZestClientElementMouseOver();
+        // Then
+        assertEquals(true, clientElementMouseOver.isEnabled());
+    }
+
+    @Test
+    void shouldSerialiseAndDeserialise() {
+        // Given
+        ZestClientElementMouseOver original =
+                new ZestClientElementMouseOver("sessionId", "type", "element");
+        original.setEnabled(false);
+        // When
+        String serialisation = ZestJSON.toString(original);
+        ZestClientElementMouseOver deserialised =
+                (ZestClientElementMouseOver) ZestJSON.fromString(serialisation);
+        // Then
+        assertEquals(deserialised.getElementType(), original.getElementType());
+        assertEquals(deserialised.getWindowHandle(), original.getWindowHandle());
+        assertEquals(deserialised.getElement(), original.getElement());
+        assertEquals(deserialised.isEnabled(), original.isEnabled());
+    }
+
+    @Test
+    void shouldMouseOverElement() throws Exception {
+        // Given
+        String htmlContent =
+                "<html><head><style>div:hover {color: red;}</style></head><body><div id=\"test-id\">Hover over me to see my color change.</div></body></html>";
+        server.stubFor(
+                get(urlEqualTo(PATH_SERVER_FILE))
+                        .willReturn(aResponse().withStatus(200).withBody(htmlContent)));
+        ZestScript script = new ZestScript();
+        ZestBasicRunner runner = new ZestBasicRunner();
+        // When
+        script.add(new ZestClientLaunch("windowHandle", "firefox", getServerUrl(PATH_SERVER_FILE)));
+        script.add(new ZestClientElementMouseOver("windowHandle", "id", "test-id"));
+        runner.run(script, null);
+        WebDriver driver = runner.getWebDriver("windowHandle");
+        WebElement element = driver.findElement(By.id("test-id"));
+        runner.removeWebDriver("windowHandle");
```

Comment:
This has no practical effect, should be replaced with `driver.quit();`, otherwise the tests leak browsers/WebDrivers.

Reply:
This change needs to be applied to the other tests being added as well.

Reply:
okay